client/lib/signup/step-actions/index.js

```diff
@@ -163,6 +162,20 @@ function getNewSiteParams( {
 	// when segment and vertical values are not sent. Check pbAok1-p2#comment-834.
 	const shouldUseDefaultAnnotationAsFallback = true;
 
+	// If the user didn't select any design, then we try
+	// to lookup the theme object based on the theme slug.
+	let selectedDesign = get( signupDependencies, 'selectedDesign', false );
+	if ( ! selectedDesign ) {
+		const designs = getAvailableDesigns( {
+			includeAlphaDesigns: false,
```

Comment:
Should we tie this to a flag too? (And the implementation in the other file)

By default it is using, but we are overriding it to false to make sure this is separate from gutenboarding. Also, I don't think `design-picker` should be tied to `gutenboarding`.

https://github.com/Automattic/wp-calypso/blob/8ecabde08f9a89d3bce370c6d0cb4b1388855739/packages/design-picker/src/utils/available-designs.ts#L72